packages/rs-drive-abci/src/config.rs

```diff
@@ -238,6 +238,7 @@ pub struct PlatformConfig {
     /// Enable tokio console (console feature must be enabled)
     pub tokio_console_enabled: bool,
 
+    // TODO: Use from_str_to_socket_address
```

Comment:
Todos

Reply:
you added this comment, irrelevant to this pr